lua/git-conflict.lua

```diff
@@ -236,10 +236,12 @@ end
 local function set_highlights(highlights)
   local current_color = api.nvim_get_hl_by_name(highlights.current, true)
   local incoming_color = api.nvim_get_hl_by_name(highlights.incoming, true)
-  local current_label_bg = color.shade_color(current_color.background, -10)
-  local incoming_label_bg = color.shade_color(incoming_color.background, -10)
-  api.nvim_set_hl(0, CURRENT_LABEL_HL, { background = current_color.background, bold = true })
-  api.nvim_set_hl(0, INCOMING_LABEL_HL, { background = incoming_color.background, bold = true })
+  local current_bg = current_color.background or 4218238
```

Comment:
Can you move this colors to constant variables with the other constants at the top so it's easy to change them without looking into functions

Reply:
ok